packages/fx-core/src/core/middleware/contextLoader.ts

```diff
@@ -26,15 +25,14 @@ import {
 import * as path from "path";
 import * as fs from "fs-extra";
 import { Middleware, NextFunction } from "@feathersjs/hooks/lib";
-import { validateProject } from "../../common";
+import { validateSettings } from "../../common";
 import * as uuid from "uuid";
 import { LocalCrypto } from "../crypto";
 import {
   GLOBAL_CONFIG,
   PluginNames,
   PROGRAMMING_LANGUAGE,
 } from "../../plugins/solution/fx-solution/constants";
-import { environmentManager } from "../environment";
 
 export const ContextLoaderMW: Middleware = async (ctx: CoreHookContext, next: NextFunction) => {
```

Comment:
minor: to be clear, we may need to also update the naming for `ContextLoaderMW` and `ConfigWriterMW` as their scope is changed.

Reply:
Yes, it will be in next PR. I didn't cover it in this PR because I would like to make the code diff be clearer to be reviewed😄